Sources/ArcGISToolkit/Components/Form/FormInputs/MultiLineTextEntry.swift

```diff
@@ -59,18 +59,21 @@ struct MultiLineTextEntry: View {
     var body: some View {
         FormElementHeader(element: element)
             .padding([.top], padding)
-        Group {
+        HStack(alignment: .bottom) {
             if #available(iOS 16.0, *) {
                 TextEditor(text: $text)
                     .scrollContentBackground(.hidden)
             } else {
                 TextEditor(text: $text)
             }
+            if isFocused && !text.isEmpty {
```

Comment:
This is an interesting detail and makes for a bit of a curious situation.

We could do a count of newlines in the string but that wouldn't be 100% reliable because a user could type a long sentence with zero return characters yet the editor would still display it over multiple lines. 
We could also diverge from the design spec and hide it above a certain length of text or just simply not provide the clear button for the multi-line text editor all together.

There're other solutions I've seen that involve rendering text invisibly and then computing line count based off the rendered size and current font.

Thoughts?

Reply:
I looked a bit after I commented and couldn't find a way to count lines of text in a TextView. I have looked at other Apple iPad apps and I don't see any uses of a "clear" button for multi-line text input, only for single line input. However, Field Maps DOES always display a clear button for multi-line text.

So I'm fine with this as it is; if there's a desire to change it in the future, we can.

Reply:
Sounds good. Soren also pointed out to me that the Android variant of the multi-line text entry design has the clear button always present when focused despite the number of lines.